sessions/react-state/demo-counter-start/README.md

```diff
@@ -0,0 +1,47 @@
+# React State: Demo Counter Start
```

Comment:
Should we empty this readme or write a single sentence to inform that this is a demo?